language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/extensions/ballerina/symbol/ExpressionRange.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2022, WSO2 Inc. (http://wso2.com) All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.ballerinalang.langserver.extensions.ballerina.symbol;
+
+import io.ballerina.tools.text.LinePosition;
+
+/**
+ * Represents a range of a construct in the source document.
+ */
+public class ExpressionRange {
```

Comment:
Can't we reuse `LineRange`? I guess it's the `filePath` field that's the issue. What if we keep it null and reuse it?

Reply:
Fixed with f8d2f6eee7fb44e6584f4520de64d82fd3d4533c